src/client/vite.config.ts

```diff
@@ -153,62 +156,50 @@ const customPreloadPlugin = () => {
   return result
```

Comment:
we might be able to clean this up now (since 3.2.0):
https://github.com/vitejs/vite/blob/v3.2.5/packages/vite/CHANGELOG.md#buildmodulepreload-options